package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "relay-web-app",
+  "productName": "Forsta Relay",
```

Comment:
Does this show up in electron?  If so we should use a product-like name e.g  `Forsta Messenger`

Reply:
electron-packager uses this value to set a name for the executable.  It was my understanding Relay is the name of the product.

Reply:
It's more the project name (internal / developers) vs a  marketing term.  If this faces all customers it should match the marketing which tends to be "Forsta Messenger" or "Forsta Secure Messenger"